### ISSUE TITLE
Running on localhost

### ISSUE THREAD

**Issue by izzyx6** [Markkop/nft-marketplace#5]
Thanks for such a wonderful repo. I followed the steps in the readme, but each time I get a couple of errors. This is one of them 

Unhandled Runtime Error
Error: call revert exception (method="fetchAvailableMarketItems()", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.5.0)

**Comment by abhinav-8** [comment]
is it solved now?

**Comment by Markkop** [comment]
It seems to be working for me. Are you keeping the terminal that you ran `npm run node` open?

**Comment by abhinav-8** [comment]
Yeah but there is some issue with json , it shows an error for metadata. image and whenever I have to approve for sale,that request goes to ur contract address, how can it be corrected?
I haven't deployed it yet on vercel , would be great if u could help.

**Comment by Markkop** [comment]
That's odd. It should be trying to get the contract addresses in your local environment.
Are you using metamask on localhost?
![image](https://user-images.githubusercontent.com/16388408/177552639-3c6c123e-fd99-4a75-aa0c-a316304b6561.png)

**Comment by izzyx6** [comment]
> 

yes I keep my terminal open. If you can give a step wise process on how it set it up on my local machine like you did yours without errors, it would be great

**Comment by izzyx6** [comment]
No it isn't I don't know why that error comes. Can you please share the
repo version you deployed to Vercel
